.github/workflows/main.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       shell: bash
 
     - name: run tests
-      run: PYTEST_BACKENDS=$BACKENDS ./ci/run_tests.sh
+      run: PYTEST_BACKENDS=$BACKENDS PYTEST_EXPRESSION="not udf" ./ci/run_tests.sh
```

Comment:
For udf I knew that none of them worked and there are a lot in `ibis/backends/tets/test_vecotrized_udf.py`. Since this `PYTEST_EXPRESSION` was already being used in `Tests SQL` it seemed easier to do it that way.

Reply:
hmm IIRC this was recently changed to NOT do it this way, e.g. we are not using the PYTEST_EXPRESSION's anymore here, .g. simply xfail in the appropriate module

Reply:
Hmm - that's still the way it is done on master https://github.com/ibis-project/ibis/blob/master/.github/workflows/main.yml#L89. Happy to xfail the module on dask though.